### PR TITLE
Fix CI Skipping and Invalid Status Error for sonots/slack-notice-action@v3.1.6

### PR DESCRIPTION
**Pull request by shutosekiguchi**
## Issue
CI is skipped, and the status of sonots/slack-notice-action@v3.1.6 is marked as skipped, but the value is invalid and causes an error.
image

## Resolution
- Updated to tolerate "skipped" status.
- Added `@vercel/ncc` package due to compatibility issues between ncc and node versions, which caused the following error. @zeit/ncc` removes old versions
   - Error log: https://github.com/shutosekiguchi/slack-notice-action/actions/runs/8386520680/job/22967047589
   - Add command: `npm install --save-dev @vercel/ncc`
   - Remove command: `npm uninstall @zeit/ncc`

### CODE REVIEW

**Comment by shutosekiguchi** [comment]
I have confirmed that notifications are sent to slack with a status of skipped. ref: https://github.com/shutosekiguchi/slack-notice-action/pull/2


image
image

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Yet Another GitHub Action to notify slack.
 | text_on_success   | any string                                                 | ''                    | `text` field on success                   |
 | text_on_fail      | any string                                                 | ''                    | `text` field on failure                   |
 | text_on_cancel    | any string                                                 | ''                    | `text` field on cancellation              |
+| text_on_skipped   | any string                                                 | ''                    | `text` field on skipped                   |
```
Comment (sonots) [inline README.md:22]: @shutosekiguchi 

> | status            | 'success' or 'failure' or 'cancelled' or 'custom'          | ''                    | Use `${{ job.status }}`.                  |

What is `skipped` status? I could not find at https://docs.github.com/ja/actions/learn-github-actions/contexts#job-context.

Comment (sonots) [reply README.md:22]: You want to support https://github.com/technote-space/workflow-conclusion-action by any chance?

> status: ${{ env.WORKFLOW_CONCLUSION }} # neutral, success, skipped, cancelled, timed_out, action_required, failure

If so, could you support `neutral` and `timed_out` and `action_required` in addition to `skipped`?
Also, you could write description about supporting workflow-conclusion-action on README.md?

**Review by sonots — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sonots) [reply README.md:22]: You want to support https://github.com/technote-space/workflow-conclusion-action by any chance?

> status: ${{ env.WORKFLOW_CONCLUSION }} # neutral, success, skipped, cancelled, timed_out, action_required, failure

If so, could you support `neutral` and `timed_out` and `action_required` in addition to `skipped`?
Also, could you write description about supporting workflow-conclusion-action on README.md?

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "@types/node": "^13.1.1",
     "@types/semver": "^6.2.0",
     "@typescript-eslint/parser": "^2.13.0",
-    "@zeit/ncc": "^0.20.5",
+    "@vercel/ncc": "^0.38.1",
```
Comment (sonots) [inline package.json:45]: Thanks. I cherry-picked at https://github.com/sonots/slack-notice-action/pull/36

File: README.md (hunk 1 above)
Comment (sonots) [reply README.md:22]: @shutosekiguchi I recognized that https://github.com/technote-space/workflow-conclusion-action is in public archive. Do you still need to use this action? Are there any alternatives?

Comment (shutosekiguchi) [reply README.md:22]: I see, we are currently considering our operational approach. Thank you.

Comment (shutosekiguchi) [reply README.md:22]: @sonots 
Thank you for letting me know. I have decided not to use `technote-space/workflow-conclusion-action` anymore. Therefore, I will close this PR. Thank you for your confirmation.

Comment (sonots) [reply README.md:22]: Okay, thanks!

**Pull request closed without merging**